.github/actions/build/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -l
+echo "Running ${GITHUB_WORKSPACE}/build.sh"
+${GITHUB_WORKSPACE}/build.sh
```

Comment:
Runs the build.sh bash script in the z390 root directory. I assume GITHUB_WORKSPACE is an environment variable set correctly.

Reply:
This is set by GitHub Actions when running your build to point at the source.